RandomSample/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Why did you use MIT instead of BSD-Clause-2?

Reply:
out of habit mostly, I guess. I'm happy with its use being very permissive. But, I will change to BSD Clause 2.